src/main/resources/com/ncc/aif/ontologies/InterchangeOntology

```diff
@@ -404,3 +412,67 @@
   rdfs:domain :PrivateData ;
   rdfs:range xsd:string ;
   rdfs:subPropertyOf owl:topDataProperty .
+
+:informativeJustification rdf:type owl:ObjectProperty ;
+  rdfs:label "informative justification" ;
+  rdfs:comment "Definitive justification for an entity or entity cluster" ;
+  rdfs:domain [
+        rdf:type owl:Class ;
+        owl:unionOf (
```

Comment:
According to https://nextcentury.atlassian.net/wiki/spaces/AIDAC/pages/513736747/M18+Evaluation+Information+Including+AIF+Restrictions+and+Hypotheses and the eval plan 4.3#11, not only are entities and clusters allowed to have informative justifications, but ALSO events and relations can have them also.  They needed to be added to the unionOf, right?

Reply:
Fixed